theme/mixins/backdrop.less

```diff
@@ -1,9 +1,13 @@
 .transparentBackdrop() {
   position: fixed;
-  top: -999px;
-  left: -999px;
-  right: -999px;
-  bottom: -9999px;
+  top: 0;
+  left: 0;
+  // calc(100vw -1) is an adjustment for FF because it adheres to w3c spec
+  // in that it includes the scrollbars. This however is problematic
+  // for us as it means there is still a small space that overflows:
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=811403
+  width: calc(100vw - 1);
+  height: 100vh;
```

Comment:
Given a fixed position element, wouldn't `right: 0; bottom: 0;` give the same result as setting `width` and `height`, but without the need to deal with the viewport units and scrollbars issue? 🤔

I could be wrong though, I'm not so familiar with the styleguide itself.

Reply:
Give it a shot Mike if you got time, be great to simplify this further if possible.